lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -2234,6 +2303,20 @@ auto FEnumType::verify(function_ref<InFlightDiagnostic()> emitErrorFn,
   return success();
 }
 
+/// Return this type with any type aliases recursively removed from itself.
+FIRRTLBaseType FEnumType::getAnonymousType() {
+  if (!this->containsTypeAlias())
+    return *this;
```

Comment:
This seems better.

Reply:
Sorry, I changed to the other way implementation for the consistency. Thank you for pointing out!